internal/venice-common/src/main/resources/avro/RecordChangeEvent.avsc

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "RecordChangeEvent",
+  "doc": "Schema which describes a record change event in a Venice store.",
+  "namespace": "com.linkedin.venice.client.change.capture.protocol",
+  "type": "record",
+  "fields":
+  [
+    {
+      "name": "previousValue",
+      "doc": "",
+      "default": null,
+      "type": [
+        "null",
+        {
+          "name": "ValueBytes",
+          "doc": "Optional field which describes the previous state of a given row.  Can be null if before image isn't enabled, or if the record was null to begin with.",
```

Comment:
If before image is diaabled, will we emit this event?

Reply:
Ah, so the idea is that we can still use the change capture client on stores which don't even have the view.  In such a case, the change capture client would just read the version topic as opposed to the change capture topic.  Change capture topic only exists for cases where before image is desired.